ROUTINE error1
#; D ^test
#; D test^test("E")  ; MyTask.NewClass2でエラーを発生させる。
main(p1="abc",halttime=0) public {
	;実行結果は 
	; select * from Task_Response.CallTask
	; select * from Task_Response.CallJob
	;
	Set tSC = ##class(Ens.Director).CreateBusinessService("Direct",.tService)
	If ($IsObject(tService)) {
		Set input=##class(Task.Request.CallJob).%New()
		Set input.JobName="Job1"
		Set input.P1="E"
		Set input.P2=halttime
		Set tSC = tService.ProcessInput(input,.output)
		zw output
	}
}